app/helpers/bgeigie_imports_helper.rb

```diff
@@ -27,4 +27,10 @@ def status_details(bgeigie_import)
       a << t(".#{k}") if v
     end
   end
+
+  def operation_button(bgeigie_import, action, text = t(".#{action}"))
```

Comment:
Looks rubocop flagged the use of `#` on a `def` line here https://travis-ci.org/Safecast/safecastapi/builds/388262191?utm_source=github_status

Might probably read better to default nil then `||=` on the first function line anyway.

Reply:
It seems to be rubocop bug. It is string interpolation, but rubocop thinks that it is a comment... Use `format` instead.